packages/block-editor/src/components/block-card/stories/index.story.js

```diff
@@ -0,0 +1,76 @@
+/**
+ * WordPress dependencies
+ */
+import { box, button, cog, paragraph } from '@wordpress/icons';
+
+/**
+ * Internal dependencies
+ */
+import BlockCard from '../';
+
+export default {
```

Comment:
It might be a good idea to match the format with other stories.

```javascript
const meta = {
	title: 'BlockEditor/BlockCard',
};

export default meta;
```

Reply:
Updated this part to match the format with other stories.